### PR TITLE
HTML content is escaped before it is sent to the server (close #127)

### LINKED ISSUE #127: Reporter failes when a failed assertion contains html markup
Reporter fails with an error in the following situation:
```js
var div1 = document.getElementById('div1');
var div2 = document.getElementById('div2');

eq(div1, div2);
```

In you compare jQuery objects `eq($(div1), $(div2));` reporter writes:
```json
expected: []
actual: []
```

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: src/client/core/utils/dom.js [hunk 1]
```diff
@@ -95,7 +96,7 @@ export function isTheSameNode (node1, node2) {
         return node1 === node2;
 }
 
-export function getElementDescription (el) {
+export function getEscapedElementDescription (el) {
```
Comment (inikulin) [inline src/client/core/utils/dom.js]: Calling it `getElementDescription` was just fine.

**Comment by inikulin** [comment]
We need regresion test for this

**Comment by inikulin** [comment]
\r-

**Comment by AlexanderMoskovkin** [comment]
The existing qunit tests were modified to this fix, is it not enough?

**Comment by inikulin** [comment]
@AlexanderMoskovkin Doesn't seem so, I can't see how HTML escaping is tested via this modifications

**Comment by helen-dikareva** [comment]
@inikulin  We check currentError.actual, currentError.expected for failed assertions. We expect escaped dom-elements' descriptions https://github.com/helen-dikareva/testcafe-phoenix/commit/96b385f54cd0981abf3f24510a66bb0dc172cdee#diff-ea383fc53abbf39a18fcd9ffa701c855R215

**Comment by helen-dikareva** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by inikulin** [comment]
ping @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**